roles/config_drive/README.md

```diff
@@ -0,0 +1,60 @@
+# config_drive
+Role to build a `NoCloud` config-drive ISO image.
+
+## Privilege escalation
+No privilege escalation required.
```

Comment:
Could you please update this, you are using become for installing packages

Reply:
Would be happy to approve/lgtm after those two issues are resolved.

Reply:
Done